test/e2e/framework/gobindata_util.go

```diff
@@ -16,22 +16,16 @@ limitations under the License.
 
 package framework
 
+//go:generate ../../../hack/update-bindata.sh
```

Comment:
would it make more sense to have this script under `test/e2e/framework`? `hack/` feels logically far away from here.